src/vcpkg/archives.cpp

```diff
@@ -174,11 +178,14 @@ namespace vcpkg
             case ExtractionType::Unknown: break;
             case ExtractionType::Nupkg: win32_extract_nupkg(tools, status_sink, archive, to_path); break;
```

Comment:
This one can also *probably* use 7zr?

Reply:
7zr only support .7z formats. I tried to use it for .tar and .gz and it didn't work on those either.

Reply:
It worked for .zip in my testing (in the perf data I showed in the other thread)

Reply:
Interesting, I thought I checked .zip as well, but I could be mistaken. I'll double check that it works on that and make the change.